### PR TITLE
Add 2020 Methodology in French

### PR DESCRIPTION
**Pull request by bazzadp**
This is liable to change some more but at the moment it's very similar to 2019 and we've committed in English to think we can do the same in the other languages if one of the French speakers is willing to review.

### CODE REVIEW

File: src/templates/fr/2020/methodology.html [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+        </p>
+
+        <p>
+        Le parcours de ces sites, réalisé par HTTP Archive en août 2020 et utilisé par le Web Almanac, a utilisé la dernière version disponible du Chrome UX Report, juin 2020 (201905), pour sa liste de sites Web. Cet ensemble de données a été publié le 14 juillet 2020 et reflète les sites Web visités par les utilisateurs de Chrome pendant le mois de mai.
+        </p>
+
+        <!-- To be translated <p>
+          There was was a 20-30% growth on websites analysed compared to the <a href="../2019/methodology#websites">number of sites analysed for the 2019 Web Almanac</a>. This increase has been analysed by <a href="{{ url_for('contributors', year=year, lang=lang, _anchor='paulcalvano') }}">Paul Calvano</a> in his <a href="https://paulcalvano.com/2020-09-29-growth-of-the-web-in-2020/">Growth of the Web in 2020</a> post.
+        </p> -->
```
Comment (bazzadp) [inline src/templates/fr/2020/methodology.html:128]: Can a native speaker translate this and remove the `<!--` and `-->` marks?

File: src/templates/fr/2020/methodology.html [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+      </p>
+
+      <p>
+        Throughout August 2020, the HTTP Archive data pipeline crawled several million websites, gathering the metadata to be used in the Web Almanac.
+      </p>
+
+      <p>
+        The data analysts began writing queries to extract the results for each metric. In total, <strong>TBC queries</strong> were written by hand! You can browse all of the queries by chapter in the <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> directory of the project's GitHub repository.
+      </p>
+    </section>-->
```
Comment (bazzadp) [inline src/templates/fr/2020/methodology.html:397]: Can a native speaker translate this whole section and remove the `<!--` and `-->` marks?

File: src/templates/fr/2020/methodology.html [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+      <p>
+        Enfin, l’équipe de rédaction a revu les chapitres pour corriger de simples erreurs grammaticales et assurer l’uniformité de l’expérience de lecture.
+      </p>
+    </section>
+
+    <section>
+      <h2 id="looking-ahead"><a href="#looking-ahead" class="anchor-link">Perspectives</a></h2>
+
+      <p>
+        L’édition 2020 du Web Almanac est la deuxième de une tradition annuelle d’introspection et d’investissement positif au sein de la communauté Web. Parvenir à ce résultat a demandé un effort monumental de la part de nombreux <a href="{{ url_for('contributors', year=year, lang=lang) }}">contributeurs et contributrices</a> et nous espérons pouvoir grandement tirer parti de ce travail pour faciliter les éditions futures.
```
Comment (bazzadp) [inline src/templates/fr/2020/methodology.html:419]: This line could do with a review

File: src/templates/fr/2020/methodology.html (hunk 2 above)
Comment (borisschapira) [reply src/templates/fr/2020/methodology.html:397]:
```suggestion
    <section>
      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">Réflexion</a></h3>

      <p>
        Le Web Almanach 2020 a été lancé en juin 2020 par un <a href="https://twitter.com/rick_viscomi/status/1273135952848977920?s=20">appel en direction de nouveaux auteurs et autrices</a>. Après une phase de réflexion, 22 chapitres ont été consolidés et nous avons commencé à <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/876">assigner des experts et expertes sur chaque sujets et des pairs examinateurs et examinatrices à chaque chapitre</a>. Ce processus souffre d’un biais inhérent à la difficulté d’obtenir l’engagement de volontaires pour un projet de cette envergure. Ainsi, de nombreuses personnes qui contribuent au projet sont membres des mêmes cercles professionnels. Un objectif explicite des futures éditions du Web Almanac est d’encourager encore plus l’inclusion de voix sous-représentées et hétérogènes en tant qu’auteurs, autrices, relecteurs et relectrices.
      </p>

      <p>
        Nous avons passé le mois de juillet 2020 à apparier les personnes avec les chapitres et à recueillir leurs commentaires pour finaliser les indicateurs particuliers qui constitueront chaque chapitre.
      </p>
    </section>

    <section>
      <h3 id="analysis"><a href="#analysis" class="anchor-link">Analyse</a></h3>

      <p>
        En juillet et août 2020, ayant une liste bien établie de mesures et de chapitres, les analystes de données ont trié les mesures pour en déterminer la faisabilité. Dans certains cas, il a fallu créer des <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">mesures personnalisées</a> pour combler les lacunes de nos capacités analytiques.
      </p>

      <p>
        Tout au long du mois d’août 2020, le pipeline de données HTTP Archive a parcouru plusieurs millions de sites web, rassemblant les métadonnées à utiliser dans le Web Almanac.
      </p>

      <p>
        Les analystes de données ont commencé à rédiger des requêtes pour extraire les résultats de chaque mesure. Au total, les <strong> requêtes TBC</strong> ont été écrites à la main ! Vous pouvez parcourir toutes les requêtes par chapitre dans le répertoire <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> du dépôt GitHub du projet.
      </p>
    </section>
```

Not sure I understood:

> In total, <strong>TBC queries</strong> were written by hand!

I would have thought a number would be provided.

File: src/templates/fr/2020/methodology.html (hunk 1 above)
Comment (borisschapira) [reply src/templates/fr/2020/methodology.html:128]:
```suggestion
        <p>
                Il y a eu une croissance de 20 à 30 % des sites web analysés par rapport au <a href="../2019/methodology#websites">nombre de sites analysés pour le Web Almanac 2019</a>. Cette augmentation a été analysée par <a href="{{ url_for('contributors', year=year, lang=lang, _anchor='paulcalvano') }}}">Paul Calvano</a> dans son article <a href="https://paulcalvano.com/2020-09-29-growth-of-the-web-in-2020/" lang="en">Growth of the Web in 2020</a>.
        </p>
```

File: src/templates/fr/2020/methodology.html (hunk 3 above)
Comment (borisschapira) [reply src/templates/fr/2020/methodology.html:419]:
```suggestion
        L’édition 2020 du Web Almanac est la deuxième d’une tradition annuelle dans la communauté du web d’introspection et d’engagement pour un changement positif. Pour en arriver là, un effort monumental a été fourni par de nombreux et dévoués <a href="{{ url_for('contributors', year=year, lang=lang) }}">contributeurs et contributrices</a> et nous espérons pouvoir grandement tirer parti de ce travail pour faciliter les éditions futures.
```

**Review by borisschapira — CHANGES_REQUESTED** [review]

File: src/templates/fr/2020/methodology.html (hunk 2 above)
Comment (bazzadp) [reply src/templates/fr/2020/methodology.html:397]: It will once we stop writing the queries! 😁 
though good reminder - added to #1404

**Comment by bazzadp** [comment]
Thanks @borisschapira for the speedy translation!

**Pull request merged**